lib/matplotlib/collections.py

```diff
@@ -415,7 +415,7 @@ def set_hatch(self, hatch):
         can only be specified for the collection as a whole, not separately
         for each member.
 
-        ACCEPTS: [ '/' | '\\\\' | '|' | '-' | '+' | 'x' | 'o' | 'O' | '.' | '*' ]
+        ACCEPTS: [ '/' | '\\' | '|' | '-' | '+' | 'x' | 'o' | 'O' | '.' | '*' ]
```

Comment:
Also, since you're going to have to push an update, I cancelled the OSX build which has been waiting for a while because we've got a long backlog.